packages/dapp/src/utils/helpers.js

```diff
@@ -210,3 +211,14 @@ export const getTokenSymbol = (token, chainId, tokenData) => {
 export const dateTimeToDate = dateTime => {
   return dateTime.split(',')[0];
 };
+
+export const getAgreementLink = projectAgreement => {
+  const address = projectAgreement[projectAgreement.length - 1].src;
+  if (projectAgreement[projectAgreement.length - 1].type === 'ipfs') {
+    const hash = address.substring(7);
```

Comment:
Now that I had you change the if statement, 7 is more of a magic number. Perhaps a comment making clear the 'ipfs://' = 7

Reply:
sounds good, added